tests/Organization.py

```diff
@@ -160,6 +160,7 @@ def testCreateTeamWithAllArguments(self):
             "push",
             "secret",
             "Description also created by PyGithub",
+            141496,
```

Comment:
Can we assert the `parent_team_id` in `testAttributes`? I may have to record replay data for this.